test/upload_validation_test.ts

```diff
@@ -4735,6 +4735,32 @@ describe('Pack metadata Validation', async () => {
       await validateJsonAndAssertFails(metadata);
     });
 
+    it('Fail when both required fields are missing for users', async () => {
+      assert.throws(() =>
```

Comment:
Probably good to add a failure message or regex as the second arg here to make sure it's throwing for the expected reason.